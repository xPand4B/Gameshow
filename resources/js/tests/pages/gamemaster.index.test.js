import TestBehaviour from '../TestBehaviour';
import Component from '../../src/pages/gamemaster/index';

describe('Gamemaster.Index', () => {
    beforeEach(() => TestBehaviour.beforeEach())

    test('component can be mounted', () => {
        const wrapper = TestBehaviour.mountFunction(Component);

        expect(wrapper.html()).toMatchSnapshot();
    });

});
