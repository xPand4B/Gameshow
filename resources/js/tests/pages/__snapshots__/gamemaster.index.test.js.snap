// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gamemaster.Index component can be mounted 1`] = `
"<div class=\\"w-100\\">
  <div class=\\"row h-60\\">
    <div class=\\"red col col-5\\">
      Management
      <ul>
        <li>Mark current player</li>
        <li>Manual Points</li>
      </ul>
    </div>
    <div class=\\"blue col\\">
      Question + Answers + Correct one
    </div>
  </div>
  <div class=\\"row h-40\\">
    <div class=\\"orange col col-12\\">
      Player over-view
    </div>
  </div>
</div>"
`;
